autotest/OldTests/src/test/java/io/github/openequella/pages/search/NewSearchPage.java

```diff
@@ -188,6 +188,41 @@ public void selectOwner(String ownerName) {
     waiter.until(ExpectedConditions.invisibilityOfElementLocated(By.className("MuiDialog-root")));
   }
 
+  /**
+   * Returns a By for the aria-label attribute of an attachment link in a search result, given the
+   * attachment title.
+   *
+   * @param attachmentText The text to search against to find the name of the attachment within
+   *     search results.
+   * @return a By for the attachmentText.
+   */
+  private By xPathForAttachmentText(String attachmentText) {
```

Comment:
Bit of an odd name there, would expect something along the lines of:

```suggestion
  private By attachmentLinkByText(String text) {
```